elasticsearch.go

```diff
@@ -37,6 +37,8 @@ type Config struct {
 	CloudID string // Endpoint for the Elastic Service (https://elastic.co/cloud).
 	APIKey  string // Base64-encoded token for authorization; if set, overrides username and password.
 
+	CACert []byte // PEM-encoded certificate authorities.
```

Comment:
I think you should also document behaviour with respect to system defaults: does it override or extend? (See further comments.)

Perhaps also mention that this only works if Transport is unspecified, or is an `*http.Transport`.

Reply:
Addressed in dcdb7f2.